client/graphics/view.hpp

```diff
@@ -6,9 +6,9 @@
 
 namespace tec {
 struct View {
-	View(bool active = false) : active(active) {}
+	explicit View(const bool active = false) : active(active) {}
```

Comment:
primitive types like `bool` don't really need const when used an argument, I even saw `const` removed from some other primitive, yet it's added here?

Reply:
My linter pinged some of these, and I just blindly accept it it would seem

Reply:
Reverted